### PR TITLE
Garage: Read zora api key from env var

### PR DESCRIPTION
**Pull request by datadanne**

### CODE REVIEW

File: garage/src/hooks/useZDK.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const API_ENDPOINT = "https://api.zora.co/graphql";
 const args = {
   endPoint: API_ENDPOINT,
   networks: [networkInfo],
-  // apiKey: process.env.API_KEY,
+  apiKey: import.meta.env.VITE_ZORA_API_KEY
```
Comment (sanderpick) [inline garage/src/hooks/useZDK.ts:12]: 👍

Comment (sanderpick) [reply garage/src/hooks/useZDK.ts:12]: can i use this for both dev and prod (mumbai and mainnet)?

**Review by sanderpick — APPROVED** [review]

File: garage/src/hooks/useZDK.ts (hunk 1 above)
Comment (datadanne) [reply garage/src/hooks/useZDK.ts:12]: Yeah we can for now. Zora's NFT api doesn't index mumbai but we don't need that feature right now, we will have to rethink the dev setup later if we want to load custom pilot NFTs

Comment (sanderpick) [reply garage/src/hooks/useZDK.ts:12]: cool, makes sense

**Comment by sanderpick** [comment]
gonna merge this one 🙏

**Pull request merged**